composer.lock

```diff
@@ -1459,8 +1382,7 @@
                 "BSD-3-Clause"
             ],
             "description": "Replace zendframework and zfcampus packages with their Laminas Project equivalents.",
-            "time": "2020-05-20T13:45:39+00:00"
-        },
```

Comment:
@ihor-sviziev @ProkopovVitaliy The tests are failing with to prepare the environment with error `./composer.lock" does not contain valid JSON, Parse error on line 1385`. Looks like it's because of this removed `},`